installing/installing_aws/installing-aws-government-region.adoc

```diff
@@ -1,12 +1,12 @@
 [id="installing-aws-government-region"]
-= Installing a cluster on AWS into a government or secret region
+= Installing a cluster on AWS into a government region
 include::modules/common-attributes.adoc[]
 :context: installing-aws-government-region
 
 toc::[]
 
 In {product-title} version {product-version}, you can install a cluster on
-Amazon Web Services (AWS) into a government or secret region. To configure the
+Amazon Web Services (AWS) into government regions. To configure the
```

Comment:
The change from "into a government [...] region" to "into government regions" diverges from the title of the page which still uses the "into a government region" language. I suggest that we should be consistent, one way or the other.